### DISCUSSION TITLE
Logistics aggregator

### DISCUSSION THREAD

**Discussion by ghost** [karrioapi/karrio#415] [General]
Hi! I am new to Karrio and logistics in general.  Can Karrio be deployed as  platform aggregator  for distributed nation-wide logistics ecosystem to support e-commerce fulfillment.

**Comment by danh91** [comment]
> Hi! I am new to Karrio and logistics in general.  Can Karrio be deployed as  platform aggregator  for distributed nation-wide logistics ecosystem to support e-commerce fulfillment.

Hi @spiliti, 

Using karrio an aggregator for a logistics ecosystem for e-commerce fulfillment is indeed one of the main use case.

The main idea is to standardize shipping data under a uniform API. The code is Open Source and modular so that you can easily extend, add custom carriers when necessary and build on top.
